cogs/Twitter.py

```diff
@@ -0,0 +1,693 @@
+        logger.info(
+            f"Generated new accounts list with {len(accounts_list)} twitter accounts to follow"
+        )
+        return accounts_list
+
+    async def streaming(self):
+        self.feed = self.client.stream.statuses.filter.post(follow=self.gen_accounts)
+
+        async with self.feed as stream:
+            self.stream_thing = stream
+            async for tweet in self.stream_thing:
+                if on_tweet_with_media(tweet):
+                    logger.info(
+                        f"stream processing @{tweet.user.screen_name} - {tweet.id_str}"
+                    )
+                    await self.manage_twt(tweet)
+
+    @auto_help
+    @commands.group(
+        name="twitter",
+        aliases=["twt"],
+        brief="Automatically post twitter pics to specific channels based on tags and accounts",
+        invoke_without_command=True,
+    )
+    async def twitter(self, ctx):
+        await ctx.send_help(self.twitter)
+
+    @twitter.command(brief="Prints Twitter Stats")
+    @commands.is_owner()
+    async def stats(self, ctx):
+        accts_followed = len(self.gen_accounts)
+        time_start = time.time()
+        await self.client.api.statuses.show.get(
+            id=1230927030163628032, include_entities=False, trim_user=True
+        )
+        time_end = time.time()
+        stats_embed = discord.Embed()
+        stats_embed.set_author(name="Twitter Cog Global Stats")
+        stats_embed.add_field(
+            name="Number Accounts Followed", value=f"{accts_followed}", inline=False
+        )
+        stats_embed.add_field(
+            name="Twitter API Latency",
+            value=f"{(time_end-time_start)*1000}ms",
+            inline=False,
+        )
+        await ctx.send(embed=stats_embed)
+
+    @twitter.command(brief="Enable Twitter pic posting on server")
+    @commands.has_permissions(administrator=True)
+    @ack
+    async def enable(self, ctx):
+        """
+        Enables Twitter pic posting on the server, can specify the default channel
+
+        Example usage:
+        `{prefix}twitter enable`
+        """
+        async with self.bot.Session() as session:
+            server_settings = await db.get_twitter_settings(session, ctx.guild.id)
+            if server_settings:
+                if server_settings[0].enabled:
+                    await ctx.send(f"Twitter already enabled for {ctx.guild.name}")
+                else:
+                    logger.info(f"{ctx.guild.name} re-enabled twitter")
+                    server_settings[0].enabled = True
+            else:
+                server_settings = TwtSetting(
+                    _guild=ctx.guild.id,
+                    default_channel=0,
+                    use_group_channel=False,
+                    enabled=True,
+                )
+                session.add(server_settings)
+                logger.info(f"{ctx.guild.name} enabled twitter")
```

Comment:
```suggestion
                logger.info(f"{ctx.guild.name} ({ctx.guild.id}) enabled twitter")
```
Best to change other occurrences too. Just the name isn't very useful